crates/typst/src/model/table.rs

```diff
@@ -464,6 +490,46 @@ impl TryFrom<Content> for TableItem {
 }
 
 /// A repeatable table header.
+///
+/// You should wrap your tables heading rows in this function even if you do not
+/// plan to wrap your table across pages because Typst will use this function to
+/// attach accessibility metadata to tables in the future and ensure universal
+/// access to your document.
```

Comment:
Missing apostrophe + small paragraph suggestion at the bottom (optional, felt like it fit with the previous warning)

```suggestion
/// You should wrap your tables' heading rows in this function even if you do not
/// plan to wrap your table across pages because Typst will use this function to
/// attach accessibility metadata to tables in the future and ensure universal
/// access to your document.
///
/// You can use the `repeat` parameter to control whether your table's header
/// will be repeated across pages.
```